terraform/spec/fixtures/projects/nested_modules/main.tf

```diff
@@ -0,0 +1,5 @@
+module "github_terraform_read_only" {
+  source  = "terraform-aws-modules/iam/aws//modules/iam-assumable-role"
```

Comment:
Is the `//` intentional?

Reply:
Yeah this seems like a convention for getting nested modules, there are some examples here: https://registry.terraform.io/modules/terraform-aws-modules/iam/aws/latest

Using `source  = "terraform-aws-modules/iam/aws/modules/iam-assumable-role"` raised an error from terraform:  `Cannot apply a version constraint to module "github_terraform_read_only" (at main.tf:9) because it has a non Registry URL.`